src/web/utils/errorHandler.scss

```diff
@@ -1,5 +1,31 @@
-.error-content {
-  align-self: flex-start;
-  text-align: center;
-  padding: 4.5rem 6.5rem;
+.app {
+  .error-content {
```

Comment:
We could probably extract this part out so we don't need to define it twice.
Something like:
```
.error-content {
  // common stuff
}

.app .error-content {
  // light mode specific stuff
}

.app.darkmode .error-content {
 // dark mode specifif stuff
}
```